Dockerfile

```diff
@@ -1,2 +1,23 @@
-FROM node:0.10-onbuild
-EXPOSE 3000
\ No newline at end of file
+FROM node:0.10
+
+RUN mkdir -p /opt/wftda/bouttime
+WORKDIR /opt/wftda/bouttime
+
+ARG NODE_ENV
+ENV NODE_ENV $NODE_ENV
+
+RUN echo '{ "allow_root": true }' > /root/.bowerrc
```

Comment:
Yup. It looks for one local the current working directory, then works its way up through the directory structure, before looking in $HOME or global. Since this config option is only for the docker build, not local, I decided to just create it on the fly.